lib/render/FlowRenderer.cpp

```diff
@@ -245,7 +245,7 @@ int FlowRenderer::_paintGL( bool fast )
         // First step is to re-calculate deltaT
         rv = _velocityField.CalcDeltaTFromCurrentTimeStep( _cache_deltaT );
         if( rv == flow::FIELD_ALL_ZERO ) {
-            MyBase::SetErrMsg("The velocity field seems to contain only zero values!");
+            MyBase::SetErrMsg("The velocity field seems to contain only zero or missing values!");
```

Comment:
One small nit: rather than saying the flow field contains "missing values", which may or may not be true, how about saying something more generic and correct such as "unusable values",  or 'unsuitable values", or simply 'invalid values'?

Reply:
Hey @clyne , I have changed to simply say "invalid values." I think it's good to go now!